lib/kernel-picker.js

```diff
@@ -1,70 +1,64 @@
 "use babel";
 
-import { SelectListView } from "atom-space-pen-views";
+import SelectListView from "atom-select-list";
 import _ from "lodash";
 
 import { log } from "./utils";
 
-// View to display a list of grammars to apply to the current editor.
-export default class SignalListView extends SelectListView {
-  initialize(kernelSpecs) {
+export default class SignalListView {
```

Comment:
I guess we could call this class `KernelPicker` right? Left as is to minimize diff for now.